src/smartbanner.js

```diff
@@ -138,6 +139,17 @@ export default class SmartBanner {
     return this.options.closeLabel !== undefined ? this.options.closeLabel : DEFAULT_CLOSE_LABEL;
   }
 
+  get buttonLabel() {
+    let buttonLabel = this.options.button;
+    if (this.platform === 'android') {
```

Comment:
For this to be backward-compliant as well as remain easy to use, we need to treat these settings as optional and check here if platform-specific label is actually provided in the meta tags as well. 

See above for the price suffix.